pages/advanced/cli-reference/tx-service-operations.mdx

```diff
@@ -0,0 +1,100 @@
+import { Callout } from 'nextra/components'
+
+# Operations for the `tx-service` mode
+
+## Return balance
+
+Returns a list of balances for ERC-20 tokens and ether.
+
+```bash
+balances
+```
+
+## Transactions
+
+### Return transaction history
+
+Returns history of multi-sig transactions (including pending).
+
+```bash
+history
+```
+
+### Execute a pending transaction
+
+Execute a pending transaction with enough signatures.
+
+```bash
+execute-tx <safe-tx-hash>
+```
+
+### Sign transaction
+
+Sign a transaction with the loaded owners for the provided `SafeTxHash`.
+
+```bash
+sign-tx <safe-tx-hash>
+```
+
+### Sign message
+
+Sign the provided string message provided by standard input or the `EIP712` provided by file.
+
+```bash
+sign_message [--eip191_message <str>] [--eip712_path <file-path>]
```

Comment:
As this is a reference, all the params in the command should be explained in detail. This should apply to all the params in this file